plugin/dnstap/dnstapio/io.go

```diff
@@ -65,36 +79,58 @@ func (dio *dnstapIO) Dnstap(payload tap.Dnstap) {
 	}
 }
 
+func (dio *dnstapIO) close() {
```

Comment:
Why do we now also have `Close()`?

Reply:
Close() closes queue and serve() returns.
close() closes connection and encoder w/o stop serve()